src/Types/index.tsx

```diff
@@ -333,12 +333,12 @@ export enum BorderType {
 }
 
 export interface TimeRange {
-  lower: string
-  upper?: string | null
-  seconds?: number
-  format?: string
-  label?: string
-  duration?: string
+  Lower: string
+  Upper?: string | null
+  Seconds?: number
+  Format?: string
+  Label?: string
+  Duration?: string
```

Comment:
Is capitalizing these guys going to affect anything in InfluxDB?

Reply:
Seems it will actually. I'm reverting this change.